src/vs/server/node/remoteAgentEnvironmentImpl.ts

```diff
@@ -96,6 +96,15 @@ export class RemoteAgentEnvironmentChannel implements IServerChannel {
 		if (profile && !this._userDataProfilesService.profiles.some(p => p.id === profile)) {
 			await this._userDataProfilesService.createProfile(profile, profile);
 		}
+		type ProcessWithGlibc = NodeJS.Process & {
+			glibcVersion?: string;
+		};
+		let isUnsupportedGlibc = false;
+		if (process.platform === 'linux') {
+			const glibcVersion = (process as ProcessWithGlibc).glibcVersion;
+			const minorVersion = glibcVersion ? parseInt(glibcVersion.split('.')[1]) : 28;
```

Comment:
Here glibcVersion might be undefined for vanilla nodejs builds. We should check for undefined.

Reply:
For undefined cases, this should still end up making `isUnsupportedGlibc` to false. This will be undefined for alpine builds based on musl, so this check was necessary there as well.